src/gen_smtp_server_session.erl

```diff
@@ -79,16 +79,7 @@
 	}
 ).
 
-%% OTP-19: ssl:ssl_option()
-%% OTP-20: ssl:ssl_option()
-%% OTP-21: ssl:tls_server_option()
-%% OTP-22: ssl:tls_server_option()
-%% OTP-23: ssl:tls_server_option()
--ifdef(OTP_RELEASE).
 -type tls_opt() :: ssl:tls_server_option().
--else.
--type tls_opt() :: ssl:ssl_option().
--endif.
```

Comment:
It's probably save to remove this conditional as testing doesn't happen for Erlang/OTP < 21 anymore.